pages/blog/software-development-workflow.mdx

```diff
@@ -0,0 +1,98 @@
+pullQuote: "Grouparoo's developer tooling enables a git workflow on your Reverse ETL project."
+image: "software-development-workflow/video.png"
+tags: [product, engineering]
+---
+
+An important trend in data engineering is the move towards the software development workflow that is the norm for more product-focused teams. This includes aspects such as config-as-code checked into git, tests and continuous integration, code reviews, staging environments, and production deployments.
+
+<Image
+  alt="Software development workflow"
+  src="software-development-workflow/workflow.png"
+  centered
+  width={690}
+  height={300}
+/>
+
+Building on our [config-as-code](https://www.grouparoo.com/blog/declarative-data-sync) approach, the Grouparoo 0.5 release now makes this workflow even easier by enabling the UI to generate this config. The UI adds much more confidence to the process by:
+
+- Introspecting your source tables/columns and your destination fields
+- Verifying credentials and previewing data
+- Working with sample profiles to iterate on nuances
+- Automatically writing configuration files to check into source control
+
+Take a look at how it works.
+
+<p className="video-responsive">
+  <iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/FCtZknaX1WA?rel=0"
+    frameBorder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  ></iframe>
+</p>
+
+## Evolution
+
+Over the last few years, data teams have invested a lot of resources into making their data warehouse the source of truth for their customer data. They've generated reports and insights that has helped their business make better decisions.
+
+In this environment, it was not always necessary to go through the whole software development workflow. Many things were ad hoc anyway. Analysts were not always collaborating. The worst-case scenario was an incorrect report. That's not great, but it's also not the end of the world.
+
+As this data got more and more useful, other teams wanted to be able to put it into action. In response to that need, we work with companies that are looking to operationalize their data warehouse. Data teams are calling this [Reverse ETL](https://www.grouparoo.com/solutions/reverse-etl) because it is writing data back to the tools the business uses.
+
+These integrations create more leverage, but the worst case scenario has escalated. A [Marketo data integration](https://www.grouparoo.com/integrations/destinations/marketo) could send a million emails to the wrong people. A more rigorous workflow is needed.
+
+## Grouparoo Process
+
+In Grouparoo, our JSON-based config files map the data from your source data warehouse to destinations like [Salesforce](https://www.grouparoo.com/integrations/destinations/salesforce), [Zendesk](https://www.grouparoo.com/integrations/destinations/zendesk), and [others](https://www.grouparoo.com/integrations).
```

Comment:
SEO might be improved with "like our Salesforce Data Integration, our Zendesk integration, and others"